index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+ <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <a href="#" class="logo">
+        <h1 class="logo">My Logo</h1>
+      </a>
+      <div class="toggle">
+        <img src="/image/Union.png" alt="" class="toggle-image">
+      </div>
+      <div class="menu">
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </div> 
+    </nav>
+  </header>
+  <main>
+    <section class="page-1">
+      <img class="image image-1" src="/image/Ellipse8.png" alt="">
+      <img class="image image-2" src="/image/Ellipse11.png" alt="">
+      <img class="image image-3" src="/image/Rectangle7.png" alt="">
+      <img class="image image-4" src="/image/Rectangle4.png" alt="">
+      <img class="image image-5" src="/image/Ellipse3.png" alt="">
+      <img class="image image-6" src="/image/Ellipse8.png" alt="">
+      <img class="image image-7" src="/image/Ellipse13.png" alt="">
+      <img class="image image-8" src="/image/Ellipse12.png" alt="">
+      <img class="image image-9" src="/image/Rectangle5.png" alt="">
+      <img class="image image-10" src="/image/Ellipse8.png" alt="">
+    <div>
+      <h1 class="p1-title">I'm Mahdi Glad to see You!</h1>
+      <p class="p1-description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] Kindly follow the provided fonts and color.

Your design
![image](https://user-images.githubusercontent.com/81932478/176911444-114cb65c-ef17-4404-a2fb-765157a64e27.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/176911550-26035c68-8570-4cdc-930f-f5de5442cd8a.png)